.github/workflows/long-running-azure.yaml

```diff
@@ -370,24 +395,22 @@ jobs:
 
           echo "*** Create workspace, group and environment for test ***"
           rad workspace create kubernetes
-          rad workspace list
-          rad group create radius-e2e
-          rad group switch radius-e2e
+          rad group create kind-radius
+          rad group switch kind-radius
 
           # The functional test is designed to use default namespace. So you must create the environment for default namespace.
-          rad env create radius-e2e --namespace default
-          rad env switch radius-e2e
-          rad workspace list
+          rad env create kind-radius --namespace default
+          rad env switch kind-radius
 
           echo "*** Configuring Azure provider ***"
-          rad env update radius-e2e --azure-subscription-id ${{ secrets.INTEGRATION_TEST_SUBSCRIPTION_ID }} \
+          rad env update kind-radius --azure-subscription-id ${{ secrets.INTEGRATION_TEST_SUBSCRIPTION_ID }} \
```

Comment:
Why did we change this environment?

Reply:
Some functional tests assumes that the environment is `kind-radius` by default.

Reply:
Can you update the functional tests to use the environment name from Environment variable? hardcoded environment doesn't seem right.

Reply:
I created an issue for this since it is out of the scope of this PR and could be more work than we think as of now. Here is the follow-up issue: https://github.com/radius-project/radius/issues/6990.

Reply:
I see. We can change functional test later. Can you please define new global env var in workflow and replace the hardcoded name(`kind-radius`) with `${{ env.TestEnvName }}`

Reply:
Added the environment variable @youngbupark :)